src/main/java/mate/academy/hibernate/relations/service/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.hibernate.relations.service.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
+    }
```

Comment:
```suggestion
```

Reply:
This option is use in the service

Reply:
DataProcessingException is for DAO layer. We use it in case of DB-related issues. Do not use it on service layer.